tools/state-copy/schemas.mjs

```diff
@@ -1,18 +1,33 @@
-// This small nodejs script will pull the schemas from mainnet and then load them onto local
+// This small nodejs script will pull the schema pallet storage from mainnet and then load it onto another chain.
+//
+// WARNING: This will move ALL storage values, so if there are some you do NOT want, use the FILTER_OUT or another setup.
 
 import { copy } from "./copy.mjs";
 
+// Set up sudo account (assumes //Alice has sudo access)
+const SUDO_URI = "//Alice";
+
+// Testnet
+// const DEST_URL = "wss://0.rpc.testnet.amplica.io";
+// const SOURCE_URL = "wss://0.rpc.testnet.amplica.io";
+
+// Localhost
+const DEST_URL = "ws://localhost:9944";
+
+// Mainnet
 const SOURCE_URL = "wss://1.rpc.frequency.xyz";
-const DEST_URL = "ws://127.0.0.1:9944";
 const STORAGE_KEY = "0xeec6f3c13d26ae2507c99b6751e19e76";
 const FILTER_OUT = [
   "0xeec6f3c13d26ae2507c99b6751e19e76d5d9c370c6c8aee1116ee09d6811b0d5", // governanceSchemaModelMaxBytes
   "0xeec6f3c13d26ae2507c99b6751e19e764e7b9012096b41c4eb3aaf947f6ea429", // palletVersion
+  // Comment this out if you INCLUDE setting the currentSchemaIdentifierMaximum from the SOURCE
```

Comment:
how does one include this?

Reply:
By commenting it out. I'm open to better wording as this is a . This is a filter list, so including it here, excludes it from the copy. Commenting in out removes it from the filter list, thus copying it.

Reply:
Yeah, that is a bit confusing. How about:
```suggestion
  // Comment this out to INCLUDE setting the currentSchemaIdentifierMaximum from the SOURCE
```

Reply:
Had to merge conflict the spec_version anyway, so set!